main.py

```diff
@@ -132,6 +71,27 @@ def move_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
+            for videoframe in self.grid.videoframes:
```

Comment:
Wouldn't using a list comprehension like: `[x for x in self.videoframes if x != videoframe]` be clearer? Your choice though, it is a preference thing.